<html><head><title>Associative Analysis</title>
<link rel="stylesheet" type="text/css" href="Rchm.css">
</head>
<body>

<table width="100%"><tr><td>assocAnalysis(diffGeneAnalysis)</td><td align="right">R Documentation</td></tr></table><object type="application/x-oleobject" classid="clsid:1e2a7bd0-dab9-11d0-b93a-00c04fc99f9e">
<param name="keyword" value="R:   assocAnalysis">
<param name="keyword" value=" Associative Analysis">
</object>


<h2>Associative Analysis</h2>


<h3>Description</h3>

<p>
assocAnalysis performs the Associative-T method of
differential gene analysis.
</p>


<h3>Usage</h3>

<pre>
assocAnalysis(bAdjusted, numSlides, ctrl, exp, sdrefgrp,
pctrl, pexpm,ctrlavjs, expmavjs, ctrlsds, expmsds, rgrplen)
</pre>


<h3>Arguments</h3>

<table summary="R argblock">
<tr valign="top"><td><code>bAdjusted</code></td>
<td>
bAdjusted is a matrix of bias adjusted
normalized microarray data.</td></tr>
<tr valign="top"><td><code>numSlides</code></td>
<td>
numSlides represents the total number
of chips in the microarray experiment.</td></tr>
<tr valign="top"><td><code>ctrl</code></td>
<td>
ctrl is the number of control chips in the
microarray experiment.</td></tr>
<tr valign="top"><td><code>exp</code></td>
<td>
exp is the number of control chips in the
microarray experiment.</td></tr>
<tr valign="top"><td><code>sdrefgrp</code></td>
<td>
standard deviation of the computed
refernce group.</td></tr>
<tr valign="top"><td><code>pctrl</code></td>
<td>
p values of the control chips on
performin ttests.</td></tr>
<tr valign="top"><td><code>pexpm</code></td>
<td>
p values of the experiment chips on
performin ttests.</td></tr>
<tr valign="top"><td><code>ctrlavjs</code></td>
<td>
averages of genes across all
control chips.</td></tr>
<tr valign="top"><td><code>expmavjs</code></td>
<td>
averages of genes across all
experiment chips.</td></tr>
<tr valign="top"><td><code>ctrlsds</code></td>
<td>
standard deviations of genes across all 
control chips.</td></tr>
<tr valign="top"><td><code>expmsds</code></td>
<td>
standard deviations of genes across all
experiment chips.</td></tr>
<tr valign="top"><td><code>rgrplen</code></td>
<td>
total number of genes of the computed
refernce group.</td></tr>
</table>

<h3>Details</h3>

<p>
AssociativeAnalysis performs the Associative-T method
of differential gene analysis.The results are displayed
in a 10 column matrix as follows
</p>
<p>
Column        Representation.
</p>
<p>
1     Gene Bank Id
</p>
<p>
2     Average Signal of the Control Chips/Channels.
</p>
<p>
3     SD of Control Chips/Channels.
</p>
<p>
4     Probability that a given gene in the Control 
Chips/Channels belongs or does not belong to
background.
</p>
<p>
5     Average Signal of the Experimental Chips/Channels.
</p>
<p>
6     SD of Experimental Chips/Channels.
</p>
<p>
7     Probability that a given gene in the Experimental
Chips/Channels belongs or does not belong to background.
</p>
<p>
8     P-value from a Student T-test.
</p>
<p>
9     P-value from an Associative T-test.
</p>
<p>
10    Ratio of mean expression values (Control/Experimental).
</p>
<p>
11    Group Number.
</p>
<p>
Group Numbers are defined as follows:.
</p>
<p>
A1      Expressed above background in both sample types,
but over-expressed on the Experimental Chips/Channels.
</p>
<p>
A2      Expressed above background in both sample types, but
over-expressed on the Control Chips/Channels.
</p>
<p>
A3      Expressed above background only on the Experimental
Chips/Channel.
</p>
<p>
A4      Expressed above background only in the Control
Chips/Channel.
</p>
<p>
0       None of the above.
</p>


<h3>Value</h3>

<p>
A matrix of 11 columns as described in the details and n 
rows where n stands for the number of genes in the
microarray dataset.</p>

<h3>Author(s)</h3>

<p>
Choudary L Jagarlamudi
</p>


<h3>References</h3>

<p>
Dozmorov I,Centola,M. An associative analysis of gene
expression array data.Bioinformatics.2003 Jan22;19(2):204-11
</p>
<p>
Knowlton N,Dozmorov I, Centola M. Microarray data Analysis
Tool box(MDAT): for normalization,adjustment and analysis
of gene expression data.Bioonformatics.
2004 Dec 12;20(18):3687-90
</p>


<h3>Examples</h3>

<pre>
#see refGroup for more details
</pre>



<hr><div align="center">[Package <em>diffGeneAnalysis</em> version 1.10 <a href="00Index.html">Index]</a></div>

</body></html>
